### PR TITLE
HW for Data Base 1

### PR DESCRIPTION
**Pull request by LilianaLukash**
Please see home work for lesson 20.
Thanks in advance

### CODE REVIEW

File: HW18andmore/02.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    last_name TEXT NOT NULL,
+    age REAL
+    )
+
+CREATE TABLE mynames(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    first_name TEXT NOT NULL,
+    last_name TEXT NOT NULL,
+    age REAL
+    UNIQUE (first_name, last_name)
```
Comment (acman) [inline HW18andmore/02.txt:13]: 👍

File: HW18andmore/01.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE mynames(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    first_name TEXT,
+    last_name TEXT,
+    age REAL
```
Comment (acman) [inline HW18andmore/01.txt]: I don't think `REAL` is the correct type for a person's age, use INTEGER instead of it

File: HW18andmore/q1.txt [hunk 3]
```diff
@@ -0,0 +1,25 @@
+INSERT INTO mynames
+    (first_name, last_name, age)
+VALUES ('Lili','L', 37)
```
Comment (acman) [inline HW18andmore/q1.txt:3]: nit: multiline `VALUES` could be used
```suggestion
VALUES ('Lili', 'L', 37),
       ('Vitalii', 'Hrebennikov', 35),
        ...
```

File: HW18andmore/q1.txt [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+INSERT INTO mynames
+    (id, first_name, last_name)
```
Comment (acman) [inline HW18andmore/q1.txt]: nit: I don't think it is a good idea to add an `id` value cause it is `AUTOINCREMENT` and is created by default

**Review by acman — CHANGES_REQUESTED** [review]
Overall looks good, but the type `REAL` is not correct for `age`

File: HW18andmore/01.txt (hunk 2 above)
Comment (LilianaLukash) [reply HW18andmore/01.txt]: Please see new commit with INTEGER for person's age

File: HW18andmore/q1.txt (hunk 3 above)
Comment (LilianaLukash) [reply HW18andmore/q1.txt:3]: changed

File: HW18andmore/q1.txt (hunk 4 above)
Comment (LilianaLukash) [reply HW18andmore/q1.txt]: agree, just wanted to try

**Review by acman — APPROVED** [review]

**Pull request merged**